spec/reek/smell_detectors/feature_envy_spec.rb

```diff
@@ -152,6 +152,36 @@ def alfa
       and reek_of(:FeatureEnvy, name: 'bravo')
   end
 
+  it 'reports calls inside lambda expressions' do
+    src = <<-EOS
+      def alfa(bravo)
+        [
+          -> { bravo.charlie },
+          -> { bravo.delta },
+          -> { bravo.echo },
```

Comment:
We shouldn't need this lambda to actually trigger FeatureEnvy, now shouldn't we? The first 2 should be enough following current FeatureEnvy semantics.

Reply:
It seemed two was not enough, but I'll check again.